src/engine/tests/Test_Engine.h

```diff
@@ -66,195 +66,195 @@ class EngineTestSuite : public CxxTest::TestSuite
     {
         TS_ASSERT_THROWS_NOTHING( delete mock );
     }
-
```

Comment:
Please un-comment these tests and see that they pass.
(In general, please always review the "diff" before requesting a review, to make sure you don't miss this kind of thing)

Reply:
got it (and fixed)